public/js/controllers/siteSettingsController.js

```diff
@@ -78,4 +79,38 @@ angular.module('madisonApp.controllers')
     $scope._setFeaturedDoc = function (id) {
       return $http.post('/api/docs/featured', {id: id});
     };
+
+    $scope.changeDocOrder = function (doc, oldOrder, direction) {
+      var newOrder = oldOrder + direction;
+      // Up, down, turn around
+      if(newOrder > $scope.featuredDocs.length || newOrder < 0) {
+        return false;
+      }
+
+      var oldDoc = $scope.featuredDocs[newOrder];
+      $scope.featuredDocs[newOrder] = $scope.featuredDocs[oldOrder];
+      $scope.featuredDocs[oldOrder] = oldDoc;
+
+      var docOrder = [];
+      for(var i in $scope.featuredDocs) {
+        docOrder.push($scope.featuredDocs[i].id);
+      }
+      docOrder = docOrder.join(',');
+      console.log(docOrder);
+
+      $http.put('/api/docs/featured', {'docs' : docOrder})
+        .then(function() {
+          // growl.info($translate.instant('success.general.save'));
+        },
+        function(error) {
+          console.log('error', error);
+          growl.error($translate.instant('errors.general.save'));
+        });
+    };
+
+    $scope.removeFeaturedDoc = function (doc) {
+      $http.delete('/api/docs/featured/'+doc.id).then(function (data) {
+        $scope.featuredDocs = data.data;
```

Comment:
Is it worthwhile to clarify that if the last featured doc is removed using this screen that a sensible default is chosen? Or maybe restrict the removal of a featured doc if there is only one so they have to pick a new one first?